server/app/api.py

```diff
@@ -865,6 +868,11 @@ def autograde(self, obj, user, data):
       if 'grade_final' in data and data['grade_final']:
         #Collect all final submissions and run grades.
         deferred.defer(autograde_final_subs, obj, user, data)
+    
+        if 'promote_backups' in data and data['promote_backups']:
+            # Force promote backups and run autograder
+            defrred.defer(promote_student_backups, obj, True, user, data)
+
```

Comment:
I agree. Ideally they'd be two different endpoints. I'll file an issue for this later - but for now I'm going to run with this design. Autopromotion for now is something that is tied into autograding.